res/skins/Tango/skin.xml

```diff
@@ -115,6 +115,38 @@
       <attribute persist="false" config_key="[EffectRack1_EffectUnit4],group_[BusTalkover]_enable">0</attribute>
     </attributes>
   </manifest>
+  <!--
+  **********************************************
+  General- Skin Colour Scheme
+  **********************************************
+  -->
+  <Schemes>
+    <Scheme>
+      <Name>Nuevo</Name>
```

Comment:
Is this definitive?

Reply:
nope, just a joke. search Tango Nuevo
the branch is not building anyway, as I need some help with includes and `std::unique_ptr<SkinContext> m_pContext;` before this ever gets merged

Reply:
Good stuff :)

You can just add a new parameter to `ColorSchemeParser::setupLegacyColorSchemes` of type `const SkinContext& context`. Then pass the context like this: `*m_pContext`.

Since `m_pContext` is a unique_ptr owned by `LegacySkinParser`, we know it won't nullptr when calling `ColorSchemeParser::setupLegacyColorSchemes` which is only called by `LegacySkinParser`.